Dockerfile

```diff
@@ -1,31 +1,27 @@
 FROM alpine:latest
 
-RUN echo "@testing http://dl-cdn.alpinelinux.org/alpine/edge/main" >> /etc/apk/repositories \
-  && apk add --no-cache \
-  git \
-  make \
-  musl-dev \
-  go
-#   paxmark bison flex texinfo gawk zip gmp-dev mpfr-dev mpc1-dev zlib-dev libucontext-dev linux-headers isl-dev gcc-gnat-bootstrap
+RUN apk add --no-cache \
+    git \
+    make \
+    musl-dev \
+    go
 
 # Configure Go
 ENV GOROOT=/usr/lib/go \
     GOPATH=/go \
     PATH=/go/bin:$PATH
 
-RUN  mkdir -p ${GOPATH}/src ${GOPATH}/bin \
-  && git clone https://github.com/opskumu/helm-wrapper $GOPATH/src/github.com/opskumu/helm-wrapper \
-  && cd $GOPATH/src/github.com/opskumu/helm-wrapper \
-  && make build \
-  && make build-linux
+COPY .  $GOPATH/src/github.com/opskumu/helm-wrapper 
+WORKDIR $GOPATH/src/github.com/opskumu/helm-wrapper
```

Comment:
I think the path could be shorter than `$GOPATH/src/github.com/opskumu/helm-wrapper` and it should be easy to recognize when used in this line: `COPY --from=0 /go/src/github.com/opskumu/helm-wrapper/bin/helm-wrapper /helm-wrapper`